src/components/SecondaryPanes/Frames/index.js

```diff
@@ -84,13 +95,15 @@ class Frames extends Component {
           frameOrGroup.id
             ? FrameComponent({
                 frame: frameOrGroup,
+                copyStackTrace: () => this.copyStackTrace(),
```

Comment:
I think this would be better if I bound the function and then just passed it in...